coba/environments/primitives.py

```diff
@@ -109,7 +109,9 @@ def __init__(self,
             **kwargs : Any additional information.
         """
 
-        if 'actions' in kwargs and 'probability' in kwargs and 'rewards' not in kwargs:
```

Comment:
This makes it difficult to pass optional arguments, e.g. something like `row.get('actions')`, as `'actions' in kwargs` is true even though the value is `None`. I've seen some other conditions like it that I'll update.